Documentation/CRDs/Cluster/network-providers.md

```diff
@@ -79,27 +79,6 @@ to or from host networking after you update this setting, you will need to
 [failover the mons](../../Storage-Configuration/Advanced/ceph-mon-health.md#failing-over-a-monitor)
 in order to have mons on the desired network configuration.
 
-## CSI Host Networking
```

Comment:
In an earlier version of my work, I kept this section but renamed it `CSI Pod Networking (unsupported)` and changed some of the first paragraph to reflect that it is an option but isn't supported. In editing, I decided to remove the section entirely. If we encounter many multus users who have issues with the prerequisites, we may want to discuss adding this section back and having some limited support (mostly, setting the plugin upgrade strategy to OnDelete) for the pod networking with multus as an alternative.